packages/sakuli-legacy/src/context/legacy-lifecycle-hooks.class.spec.ts

```diff
@@ -11,16 +11,33 @@ import {
 } from "selenium-webdriver";
 import { mockPartial } from "sneer";
 import { LegacyProjectProperties } from "../loader/legacy-project-properties.class";
-import { Project, TestExecutionContext } from "@sakuli/core";
-import { TestFile } from "@sakuli/core/dist/loader/model/test-file.interface";
-import { createPropertyMapMock } from "@sakuli/commons/dist/properties/__mocks__";
-import { createTestExecutionContextMock } from "./__mocks__";
-import Mock = jest.Mock;
-import Signals = NodeJS.Signals;
+import {
+  nodeSignals,
+  Project,
+  TestExecutionContext,
+  TestFile,
+} from "@sakuli/core";
+import {
+  createPropertyMapMock,
+  createTestExecutionContextMock,
+} from "./__mocks__";
 import { releaseKeys } from "./common/release-keys.function";
 import { Key, MouseButton } from "./common";
 import { getActiveKeys } from "./common/button-registry";
-import { nodeSignals } from "@sakuli/core/dist/node-signals";
+import Mock = jest.Mock;
+import Signals = NodeJS.Signals;
+
+jest.mock("fs", () => {
+  const originalModule = jest.requireActual("fs");
+
+  return {
+    __esModule: true,
+    ...originalModule,
+    promises: {
+      realpath: jest.fn().mockImplementation(() => "dirname"),
+    },
+  };
+});
```

Comment:
We used `testExecutionContext` and `ctx` as a variable name for TestExecutionContext, so I refactored it all to `ctx` to stay consistent. In hindsight, it would have been better to use `testExecutionContext`.

Reply:
:+1: